SPARCore/GADArticle.m

```diff
@@ -13,6 +104,7 @@ @implementation GADArticle
         article.title = [NSString stringWithFormat:@"Testarticle %i", i];
         article.authors = [NSArray arrayWithObjects:@{@"name": authorNames[i], @"email": @"addisemail.edu"}, nil];
         article.series = @"ada28c7d-a49f-11e6-b9d3-a0999b05c023";
+        //URL needs to change for each article
```

Comment:
That's an option, but it may be helpful for tests in the future? Either way, this comment is specifically the comment which says the URL needs to change for each dummy article. In the future, we need to make better dummy data, but we could probably just use JSON... Just brainstorming.